src/main/java/uk/gov/hmcts/dts/mytime/models/LeaveRequest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.dts.mytime.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDate;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class LeaveRequest {
+    @NotNull(message = "Employee ID is required")
+    Integer employeeId;
```

Comment:
left out private I think :)

Reply:
ohh yeah thank you. Missed those...now made private